components/cam/bld/configure

```diff
@@ -2778,16 +2778,21 @@ sub write_filepath
     }
 
     print $fh "$camsrcdir/cam/src/dynamics/$dyn\n";
+    print $fh "$camsrcdir/homme/src/share\n";
```

Comment:
This appears to be the problem - addition of homme/src/share was moved outside the "se" dycore block.  Intentional?